### PR TITLE
[#4604] Embed icons of youtube videos on resource landing page

### PR DESCRIPTION
**Pull request by Maurier**
#4604

image

image

### Pull Request Checklist: 
- [x] Positive Test Case Written by Dev

- [ ] Automated Testing

- [x] Sufficient User and Developer Documentation

- [x] Passing Jenkins Build

- [x] Peer Code review and approval

### Positive Test Case
1. Go to the resource landing page
2. Observe new icons added next to various elements on the page.
3. Hovering over the icons shows a tooltip.
4. Clicking the icons opens the relevant video in a new tab.

### CODE REVIEW

**Comment by hydrocheck** [comment]
| METRIC | VALUE |
|:-----:|:-----:|

[https://sonarqube.cuahsi-workstation.com:9000/dashboard?id=hydroshare-4705](https://sonarqube.cuahsi-workstation.com:9000/dashboard?id=hydroshare-4705)

**Comment by ClaraCogswell** [comment]
Looks good! As long as the icon conforms to the Youtube logo requirements I think we're good to go.

**Review by ClaraCogswell — APPROVED** [review]

File: theme/templates/resource-landing-page/citation.html [hunk 1]
```diff
@@ -2,8 +2,15 @@
...
     <span id="citation" class="page-anchor"></span>
-    <h3>How to Cite
-
+    <h3>
+      <span>How to Cite</span>
+      {% if resource_edit_mode %}
+      <a data-toggle="tooltip" data-placement="auto"
+          href="https://www.youtube.com/watch?v=jSeveop01Hk" target="_blank"
+          title='How to: Publishing'
+          class="fa fa-youtube-play icon-red no-decoration space-left" style="font-size: 20px;">
```
[the remark this answers is not in the document]
Comment (Maurier) [reply theme/templates/resource-landing-page/citation.html:11]: Good catch. I'll add a dedicated class for this icon.

**Review by devincowan — APPROVED** [review]
looks great! I thought maybe there was a reason for the inline styles that I could learn from, but the dedicated class looks good...

**Pull request merged**